src/it/xerces/src/test/java/com/jcabi/xerces/XercesSampleTest.java

```diff
@@ -56,6 +57,10 @@
      */
     @Test
     public void validatesXmlForSchemaValidity() throws Exception {
+        final int zero = 0;
```

Comment:
@tmjee no problem, let's wait for @yegor256 here.
@yegor256 Could you please clarify this case?